src/evidently/ui/api/models.py

```diff
@@ -82,9 +85,10 @@ def from_snapshot(cls, snapshot: SnapshotMetadata):
         )
 
 
-class DashboardInfoModel(BaseModel):
+@dataclass
```

Comment:
but why

Reply:
First of all we need to support `widgets` type as `List[BaseWidgetInfo]`

`BaseWidgetInfo` is dataclass and seems like we can't use dataclass as pydantic field (I've been getting errors 😝) -> the only way is to convert `DashboardInfoModel` to dataclass as well

Reply:
or convert BaseWidgetInfo into BaseModel :)